[debug] [2024-04-23T02:50:14.101Z] ----------------------------------------------------------------------
[debug] [2024-04-23T02:50:14.103Z] Command:       C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2024-04-23T02:50:14.103Z] CLI Version:   13.0.1
[debug] [2024-04-23T02:50:14.103Z] Platform:      win32
[debug] [2024-04-23T02:50:14.103Z] Node Version:  v18.19.0
[debug] [2024-04-23T02:50:14.104Z] Time:          Mon Apr 22 2024 19:50:14 GMT-0700 (Pacific Daylight Time)
[debug] [2024-04-23T02:50:14.104Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-04-23T02:50:14.107Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2024-04-23T02:50:14.314Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: unable to verify the first certificate
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-04-23T02:50:14.316Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=597143380&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2024-04-23T02:55:36.990Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2024-04-23T02:55:36.991Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2024-04-23T02:55:37.078Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------811173947844362145328466","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: unable to verify the first certificate
[debug] [2024-04-23T02:55:37.079Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (C:\Users\CraigM\AppData\Roaming\nvm\v18.16.0\node_modules\firebase-tools\lib\apiv2.js:246:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[debug] [2024-04-23T02:55:37.083Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2024-04-23T02:55:37.083Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"70d4e4e0-fd0e-48b3-90f6-6a508dc437aa"}
[debug] [2024-04-23T02:55:37.145Z] *** [apiv2] error from fetch(https://auth.firebase.tools/attest, {"headers":{},"method":"POST","body":"{\"session_id\":\"70d4e4e0-fd0e-48b3-90f6-6a508dc437aa\"}"}): FetchError: request to https://auth.firebase.tools/attest failed, reason: unable to verify the first certificate
[debug] [2024-04-23T02:55:37.308Z] FetchError: request to https://auth.firebase.tools/attest failed, reason: unable to verify the first certificate
    at ClientRequest.<anonymous> (C:\Users\CraigM\AppData\Roaming\nvm\v18.16.0\node_modules\firebase-tools\node_modules\node-fetch\lib\index.js:1501:11)
    at ClientRequest.emit (node:events:517:28)
    at ClientRequest.emit (node:domain:489:12)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at TLSSocket.emit (node:domain:489:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[error] 
[error] Error: Failed to make request to https://auth.firebase.tools/attest
[debug] [2024-04-23T03:03:41.552Z] ----------------------------------------------------------------------
[debug] [2024-04-23T03:03:41.554Z] Command:       C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2024-04-23T03:03:41.554Z] CLI Version:   13.0.1
[debug] [2024-04-23T03:03:41.554Z] Platform:      win32
[debug] [2024-04-23T03:03:41.555Z] Node Version:  v18.19.0
[debug] [2024-04-23T03:03:41.555Z] Time:          Mon Apr 22 2024 20:03:41 GMT-0700 (Pacific Daylight Time)
[debug] [2024-04-23T03:03:41.556Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-04-23T03:03:41.558Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2024-04-23T03:03:41.792Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: unable to verify the first certificate
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-04-23T03:03:41.795Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[info] i  To change your data collection preference at any time, run `firebase logout` and log in again. 
[info] 
[info] Visit this URL on this device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=81729928&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2024-04-23T03:04:05.887Z] >>> [apiv2][query] POST https://accounts.google.com/o/oauth2/token [none]
[debug] [2024-04-23T03:04:05.887Z] >>> [apiv2][body] POST https://accounts.google.com/o/oauth2/token [omitted]
[debug] [2024-04-23T03:04:05.979Z] *** [apiv2] error from fetch(https://accounts.google.com/o/oauth2/token, {"headers":{},"method":"POST","body":{"_overheadLength":543,"_valueLength":208,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------732452636792564751774700","_events":{},"_eventsCount":1}}): FetchError: request to https://accounts.google.com/o/oauth2/token failed, reason: unable to verify the first certificate
[debug] [2024-04-23T03:04:05.979Z] Token Fetch Error: FirebaseError: Failed to make request to https://accounts.google.com/o/oauth2/token
    at RetryOperation._fn (C:\Users\CraigM\AppData\Roaming\nvm\v18.16.0\node_modules\firebase-tools\lib\apiv2.js:246:31)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[debug] [2024-04-23T03:04:05.983Z] >>> [apiv2][query] POST https://auth.firebase.tools/attest [none]
[debug] [2024-04-23T03:04:05.984Z] >>> [apiv2][body] POST https://auth.firebase.tools/attest {"session_id":"ce64e6c6-b943-4330-ae3a-5ebcccf9e1fa"}
[debug] [2024-04-23T03:04:06.052Z] *** [apiv2] error from fetch(https://auth.firebase.tools/attest, {"headers":{},"method":"POST","body":"{\"session_id\":\"ce64e6c6-b943-4330-ae3a-5ebcccf9e1fa\"}"}): FetchError: request to https://auth.firebase.tools/attest failed, reason: unable to verify the first certificate
[debug] [2024-04-23T03:04:06.163Z] FetchError: request to https://auth.firebase.tools/attest failed, reason: unable to verify the first certificate
    at ClientRequest.<anonymous> (C:\Users\CraigM\AppData\Roaming\nvm\v18.16.0\node_modules\firebase-tools\node_modules\node-fetch\lib\index.js:1501:11)
    at ClientRequest.emit (node:events:517:28)
    at ClientRequest.emit (node:domain:489:12)
    at TLSSocket.socketErrorListener (node:_http_client:501:9)
    at TLSSocket.emit (node:events:517:28)
    at TLSSocket.emit (node:domain:489:12)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[error] 
[error] Error: Failed to make request to https://auth.firebase.tools/attest
[debug] [2024-04-23T03:04:24.239Z] ----------------------------------------------------------------------
[debug] [2024-04-23T03:04:24.241Z] Command:       C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2024-04-23T03:04:24.241Z] CLI Version:   13.0.1
[debug] [2024-04-23T03:04:24.241Z] Platform:      win32
[debug] [2024-04-23T03:04:24.241Z] Node Version:  v18.19.0
[debug] [2024-04-23T03:04:24.242Z] Time:          Mon Apr 22 2024 20:04:24 GMT-0700 (Pacific Daylight Time)
[debug] [2024-04-23T03:04:24.242Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-04-23T03:04:24.244Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2024-04-23T03:04:24.439Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: unable to verify the first certificate
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-04-23T03:04:24.440Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
[debug] [2024-04-23T03:05:43.067Z] ----------------------------------------------------------------------
[debug] [2024-04-23T03:05:43.068Z] Command:       C:\Program Files\nodejs\node.exe C:\Program Files\nodejs\node_modules\firebase-tools\lib\bin\firebase.js login --debug
[debug] [2024-04-23T03:05:43.069Z] CLI Version:   13.0.1
[debug] [2024-04-23T03:05:43.069Z] Platform:      win32
[debug] [2024-04-23T03:05:43.069Z] Node Version:  v18.19.0
[debug] [2024-04-23T03:05:43.070Z] Time:          Mon Apr 22 2024 20:05:43 GMT-0700 (Pacific Daylight Time)
[debug] [2024-04-23T03:05:43.070Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-04-23T03:05:43.072Z] >>> [apiv2][query] GET https://firebase-public.firebaseio.com/cli.json [none]
[info] i  Firebase optionally collects CLI and Emulator Suite usage and error reporting information to help improve our products. Data is collected in accordance with Google's privacy policy (https://policies.google.com/privacy) and is not used to identify you.
 
[debug] [2024-04-23T03:05:43.255Z] *** [apiv2] error from fetch(https://firebase-public.firebaseio.com/cli.json, {"headers":{},"method":"GET"}): FetchError: request to https://firebase-public.firebaseio.com/cli.json failed, reason: unable to verify the first certificate
[warn] !  Unable to fetch the CLI MOTD and remote config. This is not a fatal error, but may indicate an issue with your network connection. 
[debug] [2024-04-23T03:05:43.256Z] Failed to fetch MOTD FirebaseError: Failed to make request to https://firebase-public.firebaseio.com/cli.json
